core-bundle/src/Controller/AbstractBackendController.php

```diff
@@ -34,7 +34,7 @@ public function __invoke(): array
                 $this->Template->version = $GLOBALS['TL_LANG']['MSC']['version'].' '.ContaoCoreBundle::getVersion();
 
                 // Handle ajax request
-                if ($_POST && Environment::get('isAjaxRequest')) {
+                if (Input::postKeys() && Environment::get('isAjaxRequest')) {
```

Comment:
How about adding an `isPost()` method that only checks if the current request is a POST request? Most of the time this is only what we want to know and the `postKeys()` method seems to do a lot more.

Reply:
Done in 9ef89021221b951c63fa6b0b26972c189a8e51d0